tests/migration_test.go

```diff
@@ -1076,13 +1076,13 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 
 			})
 
-			// We had a bug that prevent migrations and graceful shutdown when the libvirt connection
+			// We had a bug that prevent migrations and graceful shutdown when the virtqemu connection
```

Comment:
I think this should have been left unchanged: whether you connect to libvirtd or virtqemud, it's still a libvirt connection.

Reply:
Thanks, you're right!